src/Shared/SFA.DAS.SharedOuterApi/InnerApi/Requests/PutUpsertEmployerUserAccountRequest.cs

```diff
@@ -12,7 +12,7 @@ public PutUpsertEmployerUserAccountRequest(Guid userId, string id, string email,
             _userId = userId;
             Data = new
             {
-                GovUkIdentifier = id,
+                GovIdentifier = id,
```

Comment:
Not sure, how this use to work, the expected property is "GovIdentifier" rather than GovUkIdentifier, but i have modified this property as needed

Reply:
I have a _feeling_ it didnt read that property from there and use the one from the route param instead 🤔

Reply:
👍🏻